clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.json

```diff
@@ -5111,7 +5111,7 @@
           "tooltip": {
             "msResolution": true,
             "shared": true,
-            "sort": 0,
+            "sort": 2,
```

Comment:
IIUC you are just moving some classes around. How is it possible that the generated dashboard changes?

Reply:
I suspect this might be caused by using newer version of grafanalib (earlier on we didn't specify a version which may led to non-reproducible builds). Other than that I don't have a good idea why this has happened.

Reply:
Ok, I've found the culprit - I've changed the Graph class used in the Python file fromg.Graph to one with defaults provided.